pandas/io/parquet.py

```diff
@@ -172,9 +172,25 @@ def write(
 
         table = self.api.Table.from_pandas(df, **from_pandas_kwargs)
 
+        filesystem = kwargs.pop("filesystem", None)
```

Comment:
is there a reason we are not actually exposing `filesystem` as a named keyword?

Reply:
> is there a reason we are not actually exposing `filesystem` as a named keyword?

That could also be an option. I think the problem is that right now, this is a keyword that is just passed through to the underlying engine (as we do for other engine-specific keywords as well), and `filesystem` is not actually a keyword supported by fastparquet.

Reply:
Actually, to be more correct, fastparquet supports it but names it `fs`. So we could indeed add a keyword `filesystem`, and map it to `fs` for fastparquet?

Reply:
yes this sounds reasonable